app/models/agenda.js

```diff
@@ -18,6 +18,7 @@ export default class Agenda extends Model.extend(LoadableModel) {
   @belongsTo('agendastatus', { inverse: null, async: true }) status;
   @belongsTo('agenda', { inverse: 'previousVersion', async: true }) nextVersion; // Set in agenda-approve-service, read-only here
   @belongsTo('agenda', { inverse: 'nextVersion', async: true }) previousVersion; // Set in agenda-approve-service, read-only here
+  @belongsTo('user', { inverse: 'secretaryForAgendas', async: true }) secretary;
```

Comment:
Since we don't have the implementation yet it's hard to say but we save agenda very frequently during daily work.
So if this gets changes during that, the risk of "reverting" the change due to concurrency is very real.

Does it make more sense to connect the secretary to the meeting instead?
I think this might be a miscommunication (agenda and meeting are both called "agenda" by the users)
But I don't think agenda A will have a different secretary than agenda B or C.
So we just have duplication now + if you do change the secretary last minute then only the latest agenda will have the correct secretary.

Reply:
Yes, this will be the secretary for the meeting. My mistake of not making this clear in the story